4.7.4/config/fedora_camel_toolbox.script

```diff
@@ -0,0 +1,9 @@
+feature:repo-add mvn:org.fcrepo.camel/toolbox-features/4.6.1/xml/features
```

Comment:
This should probably be using 4.7.2.
https://github.com/fcrepo4-exts/fcrepo4-vagrant/blob/master/install_scripts/fedora_camel_toolbox.script#L3

Reply:
@awoods  I usually sync the software stack versions with https://github.com/fcrepo4-exts/fcrepo4-vagrant as per Fedora 4 release. The released 4.7.4 version using 4.6.1, see https://github.com/fcrepo4-exts/fcrepo4-vagrant/blob/fcrepo4-vagrant-4.7.4/install_scripts/fedora_camel_toolbox.script. I will update the 4.7.2 along with Fedora 4.7.5 release.